src/Tribe/Metabox.php

```diff
@@ -252,7 +252,7 @@ public function ajax_ticket_edit() {
 		$ticket_id = absint( tribe_get_request_var( 'ticket_id', 0 ) );
 
 		if ( ! $ticket_id ) {
-			wp_send_json_error( esc_html__( 'Invalid Ticket', 'event-tickets' ) );
+			wp_send_json_error( sprintf( esc_html__( 'Invalid %s', 'event-tickets' ), tribe_get_ticket_label_singular( 'ajax_ticket_edit_error' ) ) );
```

Comment:
This only escapes the __ string, not the strings you are inserting via printf(). Can you follow the other examples where I've made suggestions that does it like `esc_html( sprintf( .... ) )` here elsewhere in your PR?